### DISCUSSION TITLE
Please help farmers by decreasing block size and increasing the chance to get Chia

### DISCUSSION THREAD

**Discussion by Tavanaie** [Chia-Network/chia-blockchain#419] [Ideas]
Hi
I Have become familiar with Chia project and installed the blockchain software
my plot size was low , and total network space was about  200 TIB
I have emptied my laptop to have more chance , now I have more than 250 Gib of plot size but... the network size increased to about 1500TIB
This makes me upset and I guess that other amateur farmers will have same problem
This can be easily solved , by decreasing plot size from 16 Chia to low amounts like 0.1 or 0.01 Chia
Also by rewarding farmers with chia based on their plot size and time connected.
For example if you will give 4000 chia each day ,on 1000TIB network Size , a farmer with 50 Gib plot size that is online for 10 hours in day , will take 0.083 Chia each day.
I think this will encourage more people to join Chia project

**Comment by mariano54** [comment]
Two ways that this will get fixed:
1. We are changing the consensus algorithm to one that has rewards every 19 seconds instead of every 5 minutes. This will increase the chances of getting a block for small farmers.
2. We will have pools. If you join a pool, you can earn rewards much more often. This will probably be what most "one drive" farmers will do.

**Reply by Tavanaie** [reply]
Thanks a lot, but please do these as soon as possible 
people like me joined the network when network size was just about 200 TIB and now it raised a lot , but  still no gain , this will make it harder for us in future

**Comment by BrandtH22** [comment]
These changes to consensus have been made and this discussion will be closed. Feel free to open a new discussion if any questions arise
